### PR TITLE
Awesome books: with ES6

### PR DESCRIPTION
**Pull request by bobb-Rob**
I implemented the following: 

- [ ] Created Module directory for all modules. Divide your code into modules.
- [ ] Used ES6 syntax.
- [ ] All existing functionalities of the app work without errors.
- [ ] Refactored all methods and functions to arrow functions.

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+
+// Remove Book from UI, localStorage and arrayList
+const bookContainer = document.querySelector('.booksContainer');
+bookContainer.addEventListener('click', (e) => {
+  if (e.target.classList.contains('remove-btn')) {
+    const bookId = e.target.parentElement.id; // Card holder ID traversed here
+    BookCollection.removeBook(bookId);
+    removeBookLS(bookId);
+    e.target.parentElement.remove();
+  }
```
Comment (AnselemOdims) [inline index.js:38]: - [ ] At the moment, the remove button is not working at all. Kindly ensure that the book is removed from the DOM and localStorage when the remove button is clicked

**Review by AnselemOdims — CHANGES_REQUESTED** [review]
### Hi @bobb-Rob , 
You have been doing a tremendous job so far :ok_hand:
There are some issues that you still need to work on to go to the next project but you are almost there!

## To Highlight 🎉
PR has a title and summary. :heavy_check_mark:
No linter errors :heavy_check_mark:

## Required Changes ♻️
- [ ] Please also check inline comments under the review

## Optional suggestions
_Comments with the [OPTIONAL] prefix are not a must to implement however, It's strongly recommended to work on them as they can make your code better._

## Happy coding!👏👏👏

#### Please leave any questions or comments in the PR if you have any doubts or unclarity! You can tag me using @AnselemOdims for a quicker response

**_Note: Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

File: index.js (hunk 1 above)
Comment (bobb-Rob) [reply index.js:38]: Thank you for the review. I have checked my code and even live demo, its working very well,....I can remove the book both from the DOM and in Local Storage. Kindly help review from your end.

Comment (bobb-Rob) [reply index.js:38]: @AnselemOdims Could it be your browser? How do I solve this issue?

**Comment by omar-labana** [comment]
# **Approved ✔️**

Hi @bobb-Rob

Congratulations on completing this project! 👏🏼 🎉

You have correctly implemented all of the requirements📃 correctly and without any errors⚠️ or bugs🪲.

Cheers and good luck in your next milestones! 🍀

Feel free to leave any questions or comments in the PR thread(don't forget to tag me 😄) if something is not 100% clear.

---
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**